src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -82,6 +82,20 @@
  *   &lt;property name="maximum" value="2"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class MyClass {
+ *      private String myString;...
```

Comment:
please remove "..." , examples should be compile-able, they might be weird in look and artificial, BUT they must be able to compile.

Reply:
sure. How should I indicate trailing whitespace?

Reply:
`// violation this line has trailing spaces  `

Reply:
Please update PR description and use `javac` on each example file, to make sure it pass.

Reply:
got it. thanks.

Reply:
done